utils/build-script

```diff
@@ -340,13 +340,26 @@ details of the setups of other systems or automated environments.""")
         help="The host target. LLVM, Clang, and Swift will be built for this "
              "target. The built LLVM and Clang will be used to compile Swift "
              "for the cross-compilation targets.",
-        default=StdlibDeploymentTarget.host_target())
+        default=StdlibDeploymentTarget.host_target().name)
+    targets_group.add_argument(
+        "--cross-compile-hosts",
+        help="A space separated list of targets to cross-compile host Swift "
+             "tools for. Can be used multiple times.",
+        action="append", type=arguments.type.shell_split, default=[])
+    stdlib_targets = StdlibDeploymentTarget.default_stdlib_deployment_targets()
     targets_group.add_argument(
         "--stdlib-deployment-targets",
         help="list of targets to compile or cross-compile the Swift standard "
              "library for. %(default)s by default.",
         nargs="*",
-        default=StdlibDeploymentTarget.default_stdlib_deployment_targets())
+        default=[
+            target.name
+            for target in stdlib_targets])
+    targets_group.add_argument(
+        "--build-stdlib-deployment-targets",
```

Comment:
Yes, actually I was just thinking maybe we shouldn't `concat` these? How about this for an interface:

```
--cross-compile-host linux-armv7 \
--cross-compile-stdlib-targets linux-armv7 linux-x86_64 \
--cross-compile-host linux-x86_64 \
--cross-compile-stdlib-targets linux-x86_64 linux-armv7 android-armv7 \
```

If we get a list of lists, we can match up the host with the list of stdlib targets. That would be very cool.

We already support multiple hosts in one run. I don't think there's any difference between supporting one foreign host or supporting several. It's just another spin of the loop.